retrieval_service/spanner.tests.cloudbuild.yaml

```diff
@@ -38,7 +38,19 @@ steps:
       - "DB_NAME=${_DATABASE_NAME}"
     script: |
         #!/usr/bin/env bash
-        python -m pytest datastore/providers/spanner_gsql_test.py
+        python -m pytest datastore/providers/spanner_test.py
+
+  - id: Run Spanner with pg integration tests
+    name: python:3.11
+    dir: retrieval_service
+    env: # Set env var expected by tests
+      - "DB_PROJECT=$PROJECT_ID"
+      - "DB_INSTANCE=${_SPANNER_INSTANCE}"
+      - "DB_NAME=${_DATABASE_NAME}"
```

Comment:
failed test run. Seems like we'll have to change different DB_NAME. Can perhaps do ${_DATABASE_NAME}-gsql and ${_DATABASE_NAME}-pg?